pkgs/applications/misc/pastel/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, rustPlatform, Security }:
```

Comment:
This is failing due to Security not being input on nixos. You should look how other packages do this and copy them

Reply:
Fixed.